src/chanjo2/crud/samples.py

```diff
@@ -0,0 +1,81 @@
+from typing import List, Union
+
+from chanjo2.models import pydantic_models, sql_models
+from sqlalchemy.orm import Session
+
+
+### Case utils
+def get_cases(
+    db: Session, skip: int = 0, limit: int = 100
+) -> List[pydantic_models.CaseRead]:
```

Comment:
```suggestion
) -> List[CaseRead]:
```
To keep it shorter and more readable, i.e import each individual model instead to not clutter the namespace

Reply:
I wrote it like this because that file used heavily sql_models and pydantic_models, written like I did you don't get confused by which is which. But I can leave the sql_models like this (example sql_models.Case) and do as you suggest for the pydantic ones (example: CaseRead). I wouldn't substitute both, because then it becomes confusing